3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -69,6 +69,13 @@ enrich {
       # customEndpoint = {{kinesisEndpoint}}
       # customEndpoint = ${?ENRICH_STREAMS_SOURCE_SINK_CUSTOM_ENDPOINT}
 
+      ## Optional endpoint url configuration to override aws dyanomdb endpoints,
```

Comment:
Can you please precise that this is for Kinesis checkpoints.

Reply:
done